data/json/items/gun/shot.json

```
@@ -193,8 +193,8 @@
     "copy-from": "shotgun_pump_3gun",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "6-round shotgun" },
-    "description": "This is a shotgun with capacity for 6 rounds of 12 gauge shot.",
+    "name": { "str": "Mossberg 12 gauge shotgun" },
```

Again, drop the brand name, but then I feel like this is too generic of a name - hence the original scheme.
```suggestion
    "name": { "str": "12 gauge shotgun" },
```